swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2021-01-31-preview/redhatopenshift.json

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
should be swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-01-31-preview/redhatopenshift.json